bot/cogs/defcon.py

```diff
@@ -25,18 +26,25 @@
 BASE_CHANNEL_TOPIC = "Python Discord Defense Mechanism"
 
 
+class DefconLogInfo(Enum):
```

Comment:
Naming it `Action` would be nicer, or _at the least_ just `LogInfo` (but really the former is a lot better IMO).

Reply:
That's a great idea! After inspecting closely, I've renamed it to `Action` and use it not for `send_log_info()` but for `_defcon_action()` and other actions as well, it should be more consistently now.